src/mono/msbuild/android/build/AndroidApp.props

```diff
@@ -0,0 +1,18 @@
+<Project>
+  <PropertyGroup>
+    <RuntimeIdentifier>$(TargetOS.ToLowerInvariant())-$(TargetArchitecture.ToLowerInvariant())</RuntimeIdentifier>
```

Comment:
Hmm, how did `RuntimeIdentifier` get set before for Android? It doesn't seem like it was defined explicitly in `tests.mobile.targets`, is this declaration necessary if it originates elsewhere?